src/util.py

```diff
@@ -313,9 +313,6 @@ def process_config(config):
             # Do not parse the rest of the parameters for this algorithm if it is not included
             continue
 
-        if "directed" in cur_params:
```

Comment:
Would someone ever provide an old format config file that still had the directed keyword? Do we check for this and pop it without storing it?

Reply:
added a message indicating the update and popping the param and ignoring it